lib/index.js

```diff
@@ -79,19 +79,21 @@ internals.registrationConfig = (options) => {
     return config;
 };
 
-internals.initialize = function (server, next) {
+internals.initialize = (server, next) => {
 
-    const rootCollector = internals.state(server.root).collector;
+    const collector = internals.state(server.root).collector;
+    const rootKnex = Hoek.reach(collector, 'knexGroup.knex') || null;
```

Comment:
Not anymore– now the root knex instance is the one set by either specifying `knex` during plugin registration or calling `schwifty()` on the root server.  That means it's set the same as in any plugin, on `srv.realm.plugins.schwifty.knexGroup.knex`.